automation/update_package_swift.py

```diff
@@ -0,0 +1,77 @@
+import os
+from github import Github
+import hashlib
+import requests
+
+GITHUB_REPO = "mozilla/application-services"
+XC_FRAMEWORK_NAME = "MozillaRustComponents.xcframework.zip"
+PACKAGE_SWIFT = "Package.swift"
+github_access_token = os.getenv("GITHUB_TOKEN")
+as_version = os.getenv("AS_VERSION")
+
+def get_xcframework_artifact():
+    g = Github(github_access_token)
+    repo = g.get_repo(GITHUB_REPO)
+    
+    release = repo.get_release(as_version)
+    for asset in release.get_assets():
+        if asset.name == XC_FRAMEWORK_NAME:
+            return asset.browser_download_url
+
+def compute_checksum(xc_framework_url):
+    req = requests.get(xc_framework_url)
+    return hashlib.sha256(req.content).hexdigest()
+
+def update_package_swift(xc_framework_url, checksum):
+    old_url = ''
+    old_checksum = ''
+    with open(PACKAGE_SWIFT) as fp:
+        line = fp.readline()
+        while line:
+            line = fp.readline()
+            
+            # If this is the line that has the URL to
+            # the xcframework
+            if XC_FRAMEWORK_NAME in line:
+                # We locate where the start of the
+                # URL is, we add one not to replace the
+                # quotation mark
+                url_start = line.find('"http') + 1
```

Comment:
this can be cleaned up to look for quotations like the `checksum_start`, will look less like magic